core/src/test/java/io/syndesis/core/json/StringTrimmingJsonDeserializerTest.java

```diff
@@ -27,7 +27,7 @@
 
 import org.junit.Test;
 
-import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.*;
```

Comment:
not sure this was intentionally included in the PR

Reply:
no, gonna rollback.